NEWS.md

```diff
@@ -1,5 +1,11 @@
 # recipes (development version)
 
+* Types of variables have been made granular. `"nominal"` has been split into `"ordered"` and `"unordered"` and `"numeric"` has been split into `"double"` and `"integer"`. Using `has_type("nominal")` will no longer work, and `all_nominal()` will keep old behaviour. (#993)
```

Comment:
```suggestion
* Types of variables have been made more granular. `"nominal"` has been split into `"ordered"` and `"unordered"` and `"numeric"` has been split into `"double"` and `"integer"`. Using `has_type("nominal")` will no longer work, and `all_nominal()` will keep old behaviour. (#993)
```

Reply:
> Using `has_type("nominal")` will no longer work

Is the same the case for `has_type("numeric")`, if that existed? Why this deprecation while others are superseded?

Reply:
> Using has_type("nominal") will no longer work

Appears to be wrong. It still works! WIll fix that

Reply:
I have removed the like

> Using `has_type("nominal")` will no longer work, and `all_nominal()` will keep old behaviour.

as it was leftover from some intermediate changes. `has_type("nominal")` and `all_nominal()` works exactly like before